### PR TITLE
Update the color of the GCB Pro link for WP 5.7

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Changes the color of the 'Genesis Pro' submenu item to match the color in WP Core `5.7-beta1`
### Before with WP Core `5.6`
before-link-gen-pro

### After with WP Core `5.6`
after-link-gen-pro

### After with WP Core `5.7-beta1`
the-one-with-5 7

#### Testing instructions
* Go to `/wp-admin` 
* Sanity-check the blue color of `/wp-admin` > Genesis Blocks > Genesis Pro

### CODE REVIEW

**Comment by kienstra** [comment]
Hi @lukecarbis,
When do you think we should release this PR?

It looks like `5.7` stable is about 5 weeks out.

We'll probably release https://github.com/studiopress/genesis-custom-blocks/pull/60 next week.

File: package.json [hunk 1]
```diff
@@ -108,7 +108,7 @@
     "check-licenses": "wp-scripts check-licenses --prod --gpl2",
     "build": "wp-scripts build",
     "dev": "wp-scripts start",
-    "env": "wp-env",
+    "wp-env": "wp-env",
```
Comment (kienstra) [inline package.json]: So the tests pass. It worked locally before, but it looks like it has to be `wp-env` in CircleCI.

**Comment by kienstra** [comment]
Hi @lukecarbis,
Could you please review this when you can?

Thanks!

**Review by lukecarbis — APPROVED** [review]
Looking great as always, Ryan. :)

**Comment by lukecarbis** [comment]
Couple of things:

- Let's wait until 5.7 is released before we release this change. This might mean we wait before merging this PR.
- Hopefully in 5.8 CSS variables will become available. If that's the case, we can change (again) to something like `var( --wp--core--primary )` or whatever it gets called.

**Comment by kienstra** [comment]
Thanks, @lukecarbis!

>Let's wait until 5.7 is released before we release this change. This might mean we wait before merging this PR.

Good point. I'll keep this unmerged until `5.7`.

>Hopefully in 5.8 CSS variables will become available. If that's the case, we can change (again) to something like var( --wp--core--primary ) or whatever it gets called.

Oh yeah, a variable would be really nice.

**Comment by lukecarbis** [comment]
@kienstra I'm not sure CSS variables will even make it to 5.9 so maybe it's better to merge this as is.

**Comment by kienstra** [comment]
@lukecarbis,
Thanks, I'll merge this soon!

**Review by johnstonphilip — APPROVED** [review]

**Comment by kienstra** [comment]
@johnstonphilip, thanks a lot for reviewing this!

**Pull request merged**